File: python/06-funcoes/main096.py
'''Crie um programa que tenha uma função Media(), que vai receber as 2 notas de 
um aluno e retornar a sua média para o programa principal. 


Data 10/11/2025 -> 48ª Dia
'''


def Media(nota1,nota2):

    return (nota1 + nota2)/2

nota1 = int(input('Digite a primeira nota: '))
nota2 = int(input('Digite a segunda nota: '))

resultado = Media(nota1,nota2)
print(resultado)

File: c/06-funcoes/main096.c
/*Crie um programa que tenha uma função Media(), que vai receber as 2 notas de 
um aluno e retornar a sua média para o programa principal. 


Data 10/11/2025 -> 48ª Dia
*/


#include <stdio.h>

int Media(float nota1, float nota2){

    return (nota1 + nota2)/2;
}
int main(){

    float nota1,nota2;

    printf("Digite a primeira nota: ");
    scanf("%f",&nota1);

    printf("Digite a segunda nota: ");
    scanf("%f",&nota2);

    float resultado = Media(nota1,nota2);
    printf("%.2f",resultado);
    return 0;
}